Assets/Scripts/UI/Terminal/TerminalUIManager.cs

```diff
@@ -67,6 +70,13 @@ private void TabButtonClicked(Tab tabClicked)
                 upgradesPanel.SetActive(true);
                 break;
         }
+
+        // We need to close the pilots profile panel if the player navigates away from it 
+        // without hitting the back button, as otherwise it will conflict with the other panels.
+        if (pilotsUI.pilotProfilePanel != null && pilotsUI.pilotProfilePanel.activeSelf)
+        {
+            onTabButtonClicked?.Invoke();   
```

Comment:
why dont you just make the pilot panel a child of the fleet panel? this seems like an ugly way to do this. the fleet panel should manage all of this